app/sitemap.xml

```diff
@@ -0,0 +1,242 @@
+<urlset xmlns="http://www.sitemaps.org/schemas/sitemap/0.9">
```

Comment:
You can dynamically generate it for every route, using js

Reply:
how?